mqtt/mqtt-broker/src/persist.rs

```diff
@@ -76,8 +76,8 @@ pub struct FilePersistor<F> {
     format: F,
     previous_count: usize,
 
-    /// epoch is bumped on every store to disambiguate stores in the same timestamp
-    epoch: u16,
+    /// seq is bumped on every store to disambiguate stores in the same timestamp
+    seq: u16,
```

Comment:
it's like a `version`, right?

Reply:
it's kind of like a version but it resets on every restart.